.github/workflows/release-post-merge.yml

```diff
@@ -52,14 +53,14 @@ jobs:
   create-release:
     environment: release
     name: Create Release
-    if: ${{ (github.event.pull_request.merged == true && startsWith(github.head_ref, 'release/')) || github.event.inputs.version != '' }}
     runs-on: ubuntu-latest
     env:
-      IMAGE_REPOSITORY: ${{ github.event.inputs.image_repo || 'mongodb/mongodb-atlas-kubernetes-operator' }}
+      IMAGE_REPOSITORY: ${{ github.event.inputs.image_repo }}
       RELEASE_HELM: ${{ github.event.inputs.release_helm || 'true' }}
       CERTIFY: ${{ github.event.inputs.certify || 'true' }}
       RELEASE_TO_GITHUB: ${{ github.event.inputs.release_to_github || 'true' }}
-      BRANCH: ${{ github.event.inputs.branch || 'main' }}
+      BRANCH: ${{ github.event.inputs.branch || github.head_ref || github.ref_name || 'main' }}
```

Comment:
I'm wondering if only `${{ github.event.inputs.branch || github.head_ref }}` would be enough given the events that triggers this workflow

Reply:
It might be so. I prefer to have extra fallback for now.